src/toxcore/dht_new/codec.rs

```diff
@@ -76,7 +80,8 @@ impl UdpCodec for DhtCodec {
     fn encode(&mut self, (addr, dp): Self::Out, into: &mut Vec<u8>) -> SocketAddr {
         let mut buf = [0; MAX_DHT_PACKET_SIZE];
         if let Ok((_, size)) = dp.to_bytes((&mut buf, 0)) {
-                into.extend(&buf[..size]);
+            into.extend(&buf[..size]);
+        // TODO: move from tokio-core to tokio and return error instead of panic
         } else {panic!("DhtBase to_bytes error {:?}", dp);}
```

Comment:
Anyway you have to fix indent

Reply:
It is fixed

Reply:
Aha

Reply:
TODO is about panic!, So It's indent is adjusted to else {